messages/util.md

```diff
@@ -0,0 +1,3 @@
+# error.KeywordValueNotFormattedProperly
```

Comment:
I didn't see anything using this

Reply:
Vestige of trying to use flags to supply key/value pairs. Will remove